corehq/apps/userreports/es/data_source.py

```diff
@@ -219,29 +220,33 @@ def _get_total_aggregated_results(self):
 
     @method_decorator(catch_and_raise_exceptions)
     def get_total_row(self):
-        total_results = self._get_total_aggregated_results()
+        def _clean_total_row(aggregations, aggregation_name):
+            agg = getattr(aggregations, aggregation_name)
+            if hasattr(agg, 'value'):
+                return agg.value
+            elif hasattr(agg, 'doc_count'):
+                return agg.doc_count
```

Comment:
Aggregations like sum or min use 'value' where as filtered aggregations (used in expanded columns) use doc_count. Ideally, this should use a method on the column's config to interpret the results, but that'll take a small refactor. Will add it to the list of things to clean up